dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -370,7 +378,7 @@ def _collect_aggregations(
                 )
 
             # Finally, extract the output column name
-            output_col = str(inputs[0].column_name(rel))
+            output_col = expr.toString()
```

Comment:
Out of curiosity, to any of the PyExpr methods like `get_column_name` return the correct results, or do we have to rely on the stringified name for the Expr?

Reply:
Yeah the `column_name` function just returns the input column